### PR TITLE
Tickets/sitcom 1401

### PR DESCRIPTION
**Pull request by JackieS-NL**
**Introduction to GIS** reviewed by Franco Colleone. 

In Safety/Safety-Systems/ 

Origin branch (may have to change) SITCOM-1402

### CODE REVIEW

File: Safety/Safety-Systems/GIS.rst [hunk 1]
```diff
@@ -1,6 +1,325 @@
...
+
+
+The GIS General Overview (home), allows you to see the general status of the safety areas, 
+you can select one in the navigation bar to the left.
+
+Some area names are straightforward, while others are:
+
+* **AcFiEa**: Access Control System / Fire Main Controller / Earthquake Control System. The safety gates to the pier intermediate level and to level 8 are referred as access control system.
+
+* **M2CAM**: M2 area, includes the M2 actuator, the M2 hexapod, the Camera Hexapod and the Camera Rotator.
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:174]: What do you mean by M2 actuator?

Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:174]: M2 Hexapod starting with capital H as the others?

File: Safety/Safety-Systems/GIS.rst [hunk 2]
```diff
@@ -1,6 +1,325 @@
...
+
+* **PFLOW/AUX**: Pflow Controller & Auxiliary Elements.
+
+* **AUX IS**: Auxiliary Telescope Controller IS.
+
+
+You can access the **GIS Detections and Actions** section with :guilabel:`Det-Act` button
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:181]: Finish sentence with a . or :?

File: Safety/Safety-Systems/GIS.rst [hunk 3]
```diff
@@ -1,6 +1,325 @@
...
+
+.. figure:: ./_static/GIS-detections-actions.png
+      :width: 700px
+
+      GIS Detections and actions
+
+The status of the system is displayed, the symbols and their representation are: 
+
+* BYP within an orange square: Indicates the action is bypassed. It logs alarms but doesn't trigger any action.
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:190]: Perhaps highlight BYP in bold or italics?

File: Safety/Safety-Systems/GIS.rst [hunk 4]
```diff
@@ -1,6 +1,325 @@
...
+Bypass and reset actions
+-------------------------
+
+* Any user can :guilabel:`RESET` a subsystem.
+
+* **Wired subsystems** are reset in GIS Main Panel. 
+    E.g M2 interlock can be reset in GIS.
+
+* **Network based systems** reset on their own in GIS main panel after the reset in done in the specific subsytem.
+     E.g. M1M3 interlock should be reset in M1M3 IS and then it will be clear in GIS main panel.
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:216]: Below you have M1M3IS (together), I think that's how it's written.

File: Safety/Safety-Systems/GIS.rst [hunk 5]
```diff
@@ -1,6 +1,325 @@
...
+
+Bypass and reset actions
+-------------------------
+
+* Any user can :guilabel:`RESET` a subsystem.
+
+* **Wired subsystems** are reset in GIS Main Panel. 
+    E.g M2 interlock can be reset in GIS.
+
+* **Network based systems** reset on their own in GIS main panel after the reset in done in the specific subsytem.
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:215]: Consider replacing "after" by "once"

File: Safety/Safety-Systems/GIS.rst [hunk 6]
```diff
@@ -1,6 +1,325 @@
...
+
+* Only admin user can bypass a subsystem.
+
+* Detections can be bypassed but actions cannot.
+
+* The D2 earthquake alarm may be bypassed at times, as it can cause the TMA to lose power if there are power or capacitor bank issues. (The acceleration threshold is approximately 0.3g, equivalent to a magnitude 5-6 earthquake).
+
+.. note::
+    
+    D4 GIS E-stop: Safety gates E-stops are hit very often, so there were physically bypassed. This will be undone when operations start.
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:226]: "so **they** are currently physically bypassed"

File: Safety/Safety-Systems/GIS.rst [hunk 7]
```diff
@@ -1,6 +1,325 @@
...
+
+The GIS is composed of a **Main Controller** (cabinet on level 2) and **several cabinets**. The Main Controller manages all other cabinets, and user interaction is primarily through the Main Controller interface. 
+
+.. _Safety-Systems-GIS-connection-type:
+
+.. admonition:: The connection to the subsystems is by: 
+
+    * **Network**: TMA, Dome, M1M3, AuxTel.
+
+    * **Signals wires**: Earthquake, Access control, Pflow lift, Calibration Laser, and the top-end subsystems (M2Cam, these are M2, M2Hex, Camera Rotator, Camera Hexapod).
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:69]: "M2Cam that includes M2..."

File: Safety/Safety-Systems/GIS.rst [hunk 8]
```diff
@@ -1,6 +1,325 @@
...
+ * - D20 M2 mirror in *closed-loop*
+   - *Auto reset when M2 is in *closed-loop* again
+
+
+.. admonition:: Safety gates operation
+
+    * To open a gate: Press :guilabel:`release` button (gate is open), then move the handle.
+    * To close a gate: move the handle to close the gate. Press the :guilabel:`lock` button (the gate is closed).
+
+.. admonition:: A note on M1M3 IS interface
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:307]: Check wether M1M3IS is written together or two separate words.

File: Safety/Safety-Systems/GIS.rst [hunk 9]
```diff
@@ -1,6 +1,325 @@
...
+ * - D19 Man-lift parked
+   - *Not currently used (July, 2024)*
+ * - D20 M2 mirror in *closed-loop*
+   - *Auto reset when M2 is in *closed-loop* again
+
+
+.. admonition:: Safety gates operation
+
+    * To open a gate: Press :guilabel:`release` button (gate is open), then move the handle.
+    * To close a gate: move the handle to close the gate. Press the :guilabel:`lock` button (the gate is closed).
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:305]: You may capitalize Move here as it's a complete sentence.

**Review by isotuela — CHANGES_REQUESTED** [review]
Hi Jacqueline! 
Excellent work. Thank you for putting all this together. 

The structure is great and organization is clear. I am only missing instruction on how to reset from the GIS panel, for instance, D4. I can't see the RESET button in the main panel, don't you have to navigate to the Detection Page? Could you please add that information and a screenshot, perhaps in the "Bypass and Reset Actions" section?  

I have also added some minor suggestions for your consideration. 

We shall also merge this into `main` directly, for that, you'll need to rebase, and squash your commits. Let me know if you need help. 

Thank you so much!

File: Safety/Safety-Systems/GIS.rst (hunk 1 above)
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst:174]: Is the name given to that section, not sure if it has a meaning, I will ask

File: Safety/Safety-Systems/GIS.rst (hunk 8 above)
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst:307]: I read it as M1M3 IS, fixed.

File: Safety/Safety-Systems/GIS.rst (hunk 9 above)
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst:305]: Fixed, I also capitalized the buttons Release and Lock as in the AuxTel gate are written as such.

**Comment by JackieS-NL** [comment]
Ioana, I made your corrections. I added an image of the M2CAM window in the GIS for resetting the E-stop (is under the Top-End-Subsystem title). I improved the text and added a section title for the table. Fixed some labels too.

I squashed my commits and I would like your help doing the rebase.

File: Safety/Safety-Systems/GIS.rst [hunk 10]
```diff
@@ -1,6 +1,342 @@
...
+
+
+.. _Safety-Systems-GIS-Troubleshooting-Dealing-with-interlocks-in-GIS-Main-Control-Panel-Top-end-systems:
+
+Top End systems
+^^^^^^^^^^^^^^^^^
+
+Top End systems** (M2CAM), are a special case. The status listed does NOT indicate the status of each individual element, but rather the **status of the E-Stop circuit** (including reset action) for each element.
+
+After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons
```
Comment (isotuela) [inline Safety/Safety-Systems/GIS.rst:262]: I think I am confused with the E-stop status notation as I don´t see it in the screen. Do you mean the M2 ACTUATOR, M2 HEXAPOD... RESET buttons?

**Review by isotuela — CHANGES_REQUESTED** [review]
Thank you Jacqueline! 
I think I am still confused about the TEA section, with the screenshot attached (where I can´t see the E-stop) and the mentions to E-stop. 
Also, I think we must clearly include how to RESET the Wired Subsystems, even if obvious. 
Thank you so much for all your work!
One small note: Rebasing creates an additional commit, so it´s better to squash at the end, so you don´t have to squash twice.

File: Safety/Safety-Systems/GIS.rst (hunk 1 above)
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst]: It refers to the actual actuators of M2

File: Safety/Safety-Systems/GIS.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst:262]: That is the one I'm referring, the text was not clear. This is the modified version. 

"After an **STO or power cycle** of the Top End systems, you must :guilabel:`RESET` the GIS E-stop status buttons in the M2CAM window, shown in the figure below, in order to enable the system. 
"

File: Safety/Safety-Systems/GIS.rst [hunk 11]
```diff
@@ -1,6 +1,342 @@
...
+
+
+.. _Safety-Systems-GIS-Main-Controller-Interface:
+
+Bypass and reset actions
+-------------------------
+
+* Any user can :guilabel:`RESET` a subsystem.
+
+* **Wired subsystems** are reset in GIS Main Panel.
```
[the remark this answers is not in the document]
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst:215]: I don't understand, do you mean near the left edge of the window?

In the M2CAM image under Top End System title, it is shown the reset button.

File: Safety/Safety-Systems/GIS.rst [hunk 12]
```diff
@@ -69,9 +69,10 @@ The Main Controller manages all other cabinets, and user interaction is primaril
 
     * **Network**: TMA, Dome, M1M3, AuxTel.
 
-    * **Signals wires**: Earthquake, Access control, Pflow lift, Calibration Laser, and the top-end subsystems (M2Cam that includes M2, M2Hex, Camera Rotator, Camera Hexapod).
+    * **Signals wires**: Earthquake, Access control, Pflow lift, Calibration Laser, and the top-end subsystems: M2Cam (that includes M2, M2Hex, Camera Rotator, Camera Hexapod).
```
[the remark this answers is not in the document]
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst]: agreed, done

File: Safety/Safety-Systems/GIS.rst [hunk 13]
```diff
@@ -178,76 +179,106 @@ Some area names are straightforward, while others are:
...
 
-You can access the **GIS Detections and Actions** section with :guilabel:`Det-Act` button.
+The **GIS Detections and Actions** pane, accessed via the :guilabel:`Det-Act` button, 
+displays all *detections* and *actions* across all systems, making it the most useful for monitoring.
 
 .. figure:: ./_static/GIS-detections-actions.png
       :width: 700px
 
-      GIS Detections and actions
+      GIS Detections and Actions pane
```
[the remark this answers is not in the document]
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst]: I changed 'pane' to 'window'.

File: Safety/Safety-Systems/GIS.rst (hunk 10 above)
[the remark this answers is not in the document]
Comment (JackieS-NL) [reply Safety/Safety-Systems/GIS.rst]: for the records, instructions were added with images for reference

**Comment by JackieS-NL** [comment]
All corrections done, I rebased with main and I squashed the commits into one.

**Review by isotuela — APPROVED** [review]
Thank you for all the changes! 
Ioana

**Pull request merged**